src/components/ApiTokenNavbarItem/index.tsx

```diff
@@ -19,31 +28,36 @@ const ApiTokenNavbarItem = () => {
     <DropdownMenu.Root>
       <DropdownMenu.Trigger className='navbar__item navbar__link' asChild>
         <Button type='button' color={'tertiary'}>
-          {currentToken ? `Selected Token: ${currentToken.display_name}` : 'Please create a token'}
+          {currentToken ? `Selected Token: ${currentToken.display_name}` : <CreateToken />}
         </Button>
       </DropdownMenu.Trigger>
 
-      <DropdownMenu.Portal>
-        <DropdownMenu.Content
-          className={styles.DropdownMenuContent}
-          sideOffset={20}
-          alignOffset={-50}
-          align={'end'}
-        >
-          {tokens.map((tokenItem) => (
-            <DropdownMenu.Item
-              key={tokenItem.token}
-              className={styles.DropdownMenuItem}
-              onSelect={() => {
-                onSelectToken(tokenItem);
-              }}
-            >
-              {`${tokenItem.display_name}`}
+      {currentToken && (
+        <DropdownMenu.Portal>
+          <DropdownMenu.Content
+            className={styles.DropdownMenuContent}
+            sideOffset={20}
+            alignOffset={-50}
+            align={'end'}
+          >
+            {tokens.map((tokenItem) => (
+              <DropdownMenu.Item
+                key={tokenItem.token}
+                className={styles.DropdownMenuItem}
+                onSelect={() => {
+                  onSelectToken(tokenItem);
+                }}
+              >
+                {`${tokenItem.display_name}`}
```

Comment:
Why do you need to use template literals here?
can we use `{tokenItem.display_name}`